Lib/bdb.py

```diff
@@ -323,6 +438,10 @@ def clear_all_file_breaks(self, filename):
         del self.breaks[filename]
```

Comment:
I hadn't changed any code, so I wasn't sure about this when I saw it.  I've now added None to all the similar set_* functions that had another return and no return None.  Question -- if there are no returns, should I add return None?

Another question: there is some code like this - "if self.quitting: raise BdbQuit".  Should I put that on two lines?